src/Buildalyzer/Environment/BuildEnvironment.cs

```diff
@@ -124,25 +137,5 @@ public sealed class BuildEnvironment
             }
             return null;
         }
-
-        /// <summary>
-        /// Clones the build environment with a different set of build targets.
-        /// </summary>
-        /// <param name="targets">
-        /// The targets that should be used to build the project.
-        /// Specifying an empty array indicates that the <see cref="ProjectAnalyzer"/> should
-        /// return a <see cref="Microsoft.Build.Execution.ProjectInstance"/> without building the project.
-        /// </param>
-        /// <returns>A new build environment with the specified targets.</returns>
-        public BuildEnvironment WithTargetsToBuild(params string[] targets) =>

```

Comment:
What's the thinking behind removing this clone method - was it a mistake or am I missing the reason it needed to come out?